apps/api/app/agents/memory/skill_learning/storage.py

```
@@ -0,0 +1,361 @@
+
+        # Generate a unique ID
+        skill_id = str(uuid4())
+
+        # Add to ChromaDB
+        await collection.aadd_documents(
+            documents=[doc],
+            ids=[skill_id],
+        )
+
+        logger.debug(f"[{skill.agent_id}] Stored skill: {skill.trigger[:50]}...")
+        return skill_id
```

Ruff (0.14.14)

Consider moving this statement to an `else` block

(TRY300)